doc/language_reference/language_reference.md

```diff
@@ -409,9 +411,11 @@ they produce results of type `string`, e.g.:
 Other terms:
 
 ```EBNF
+vec_literal  :: "[" expr ("," expr)* "]"
```

Comment:
I think you should allow empty vectors in this grammar. 
The fact that the implementation may not is a bug.